src/vue/modules/asset-explorer/_api.js

```diff
@@ -0,0 +1,22 @@
+import { ApiCaller } from '@tokend/js-sdk'
+import { config } from './_config'
+
+let _instance = null
+
+/**
+ * @param {Wallet} wallet - wallet to sign the requests
+ */
+export function initApi (wallet) {
+  const horizonURL = config().horizonURL
```

Comment:
not so clear that horizonUrl is extracted from the config, but walltet from the arguments. how about providing both of them as arguments (via object)?